### PR TITLE
Video advanced: Set current time from progress bar

### PR DESCRIPTION
**Pull request by larsjohnsen**
The circle appears on progress bar `hover`:
image

### CODE REVIEW

File: lib/core/templates.html [hunk 1]
```diff
@@ -327,22 +327,25 @@ <h3 class="categoryButton" data-category="{{name}}">{{name}}</h3>
 					<div class="video-advanced-progress">
 						{{#advancedControls}}
 						<div class="video-advanced-position"></div>
+						<div class="video-advanced-position-cursor"></div>
```
Comment (andytuba) [inline lib/core/templates.html]: My favorite name for this control is `thumb` :grin:

Comment (allthefoxes) [inline lib/core/templates.html]: You mean :+1:  ? :joy:

File: lib/css/modules/_showImages.scss [hunk 2]
```diff
@@ -317,6 +319,23 @@ div.madeVisible {
 			padding: 0 1%;
 			box-sizing: border-box;
 
+			.video-advanced-position-cursor {
+				display: none;
+				position: relative;
+				top: -9px;
+				height: 16px;
+				width: 16px;
+				margin-left: calc(-1% + -9px);
```
Comment (erikdesjardins) [inline lib/css/modules/_showImages.scss]: This `calc` is a bit strange, it seems to work fine with `margin-left: -12px;`. Is there something I'm missing?

Comment (larsjohnsen) [inline lib/css/modules/_showImages.scss]: The width of `.video-advanced-progress` was 98% (by `padding: 0 1%`), necessitating `calc`. Updated now by removing `padding`.

**Pull request merged**